ddp-workspace/projects/ddp-sdk/src/lib/components/address/addressEmbedded.component.ts

```diff
@@ -42,6 +41,7 @@ interface ComponentState {
   formErrorMessages: string[];
   formWarningMessages: string[];
   fieldErrors: AddressError[];
+  isTemporarilyDisabled: boolean;
```

Comment:
New piece of state. We will try to make the form readonly while we do stuff. Separate but related from readonly because activity is readonly.